client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -823,10 +823,9 @@ public void testReindex() throws Exception {
             // tag::reindex-request-conflicts
             request.setConflicts("proceed"); // <1>
             // end::reindex-request-conflicts
-            // tag::reindex-request-typeOrQuery
```

Comment:
@jtibshirani  Thanks Julie. The only reason I added these changes into this PR is because `request.setSourceDocTypes("_doc");` underneath uses a query with a type that we are deprecating in this PR, and if I don't remove this line, `CRUDDocumentationIT. :: testReindex` fails.

Reply:
Thanks, I missed that!